$MirOS: ports/www/lynx/patches/patch-configure_in,v 1.3 2009/08/09 15:46:30 tg Exp $

	#1: Fix -Werror builds; suggested for upstream inclusion.
	#2: Fix spelling; suggested for upstream inclusion.

--- configure.in.orig	Mon Jun 13 00:18:30 2011
+++ configure.in	Sat Mar  3 18:20:29 2012
@@ -199,7 +199,7 @@ AC_MSG_RESULT($with_warnings)
 if test "$with_warnings" = "yes"
 then
 	CF_GCC_ATTRIBUTES
-	CF_GCC_WARNINGS
+	CF_GCC_WARNINGS(Wno-cast-qual Wno-cast-align)
 fi
 fi
 
@@ -452,7 +452,7 @@ AC_ARG_WITH(gnutls,
 	[cf_cv_use_libgnutls=no])
 ])
 
-# this option is mainly for comparing with/without Lynx's wrapper for GNUTLS.
+# this option is mainly for comparing with/without Lynx' wrapper for GNUTLS.
 AC_CACHE_CHECK(if you want gnutls-openssl compat,cf_cv_gnutls_compat,[
 AC_ARG_ENABLE(gnutls-compat,
 	[  --enable-gnutls-compat  link with gnutls-openssl compat],
